common/src/claims.rs

```diff
@@ -53,6 +53,9 @@ pub enum Scope {
     /// Create and delete projects
     ProjectWrite,
 
+    /// Perform Pro tier action
```

Comment:
Perhaps something like this?
```suggestion
    /// Create more projects than the free tier default
```